src/App.js

```diff
@@ -1,25 +1,40 @@
 import logo from './logo.svg';
+import React, { Component } from 'react';
 import './App.css';
+import movieData from './data'
+import MovieContainer from './MovieContainer'
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends Component {
+  constructor() {
+    super()
+    this.state = movieData
+  }
+
+  render() {
+    console.log(this.state)
+    return (
+      <main className="App">
+        <h1>Rancid Tomatillos</h1>
+        <MovieContainer movies={this.state.movies} />
+      </main>
```

Comment:
As this render method grows, we may end up pulling these JSX elements out into their own functions